addon/components/paper-button.js

```diff
@@ -7,14 +7,22 @@ import ColorMixin from 'ember-paper/mixins/color-mixin';
 const { computed } = Ember;
 
 export default BaseFocusable.extend(RippleMixin, ProxiableMixin, ColorMixin, {
-  attributeBindings: ['type'],
   type: 'button',
```

Comment:
Yes. Seems better to create the correct type attribute rather than create the wrong one and circumvent it.